libusb/os/windows_winusb.h

```diff
@@ -45,9 +45,13 @@
 // http://msdn.microsoft.com/en-us/library/ff545978.aspx
 // http://msdn.microsoft.com/en-us/library/ff545972.aspx
 // http://msdn.microsoft.com/en-us/library/ff545982.aspx
+// These GUIDs are provided by usbiodef.h from the Windows SDK, but use of that header by non-msvc
```

Comment:
Maybe instead of "the Windows SDK" say "the SDK" or system libraries, since "the Windows SDK" sounds like excluding MinGW, or do I get that wrong?

Reply:
I refer to this: https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/ if mingw provides a different version of these same header + lib (+ other..) files, I'm not familiar with it.

Reply:
Exactly, what I suspected, you meant the Microsoft "Windows SDK"(TM). Which you don't use if you build with MinGW.